### DISCUSSION TITLE
Which is preferred for build/test automation - npm scripts or gulp?

### DISCUSSION THREAD

**Discussion by ghost** [finos/morphir#58] [General]
The morphir-elm tree automates some tasks using npm scripts in `package.json`, and others in `gulpfile.js`.

Is there a reason to choose one over the other when adding new integration tests? Gulp.js seems the more powerful option, but existing integration tests are npm scripts.

**Comment by AttilaMihaly** [comment]
We started out with just scripts in `package.json` and moved to `gulp` later. In the process we left some tasks in the original place because running commands seemed more trivial using scripts. So the first preference would be to add new tasks to `gulp`. Hopefully we can move the existing scripts into `gulp` soon.

**Reply by AttilaMihaly** [reply]
I created an issue to keep track of this: https://github.com/finos/morphir-elm/issues/517
